management/commands/re-synch.py

```diff
@@ -240,3 +257,16 @@ def validate_response(response):
             "Eprints request failed with code %s",
             response.status_code
         )
+
```

Comment:
The three function at the bottom of this module all should have two new lines between them as they are outside the class.